docs/configuration-reference/components/headlamp.md

```diff
@@ -0,0 +1,69 @@
+# Headlamp configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+[Headlamp](https://github.com/kinvolk/headlamp) is an easy-to-use and versatile
+dashboard for Kubernetes.
+
+It has a clean and modern UI, it is vendor independent, generic, and supports
+the most common operations for Kubernetes clusters.
+
+## Prerequisites
+
+* A Lokomotive cluster accessible via `kubectl`.
+
+* An ingress controller such as [Contour](contour.md) for HTTP ingress.
+
+* A certificate manager such as [cert-manager](cert-manager.md) for valid certificates.
```

Comment:
I think we require `cert-manager` specifically here, as we use `cert-manager.io/cluster-issuer` annotation.